README.md

```diff
@@ -10,19 +10,60 @@ An Express REST API that mocks a minimal implementation of a centralized exchang
 
 ## Getting Started
 
+First, clone this repository locally and move into it:
 ```
-# start a postgres db
-docker run \
-  -p 5432:5432 \
-  --name portal-exchange-db \
-  -e POSTGRES_PASSWORD=password \
-  postgres
 
-# install dependencies
-yarn
+git clone https://github.com/portal-hq/portalex.git && cd portalex
 
-# start the server
-CUSTODIAN_API_KEY=<YOUR_CUSTODIAN_API_KEY> \
-  WEBHOOK_URL=<YOUR_WEBHOOK_URL> \
-  yarn demo
 ```
+
+Then check if you have anything running on ports 3000, 5432, and 4040:
+```
+
+sudo lsof -i :3000
+sudo lsof -i :5432
+sudo lsof -i :4040
+
+```
+
+If it states that there is something running on those ports, use the command "kill -9 <Port ID (PID)>" so that no error arises.
+
+Now that you know there won't be any port-related errors, create and enter a ".env" file:
+```
+
+touch .env && nanon .env
+
+```
+
+This should open up the ".env" file, where you should copy, paste, and replace all of the following info:
```

Comment:
a good convention for `.env` file is to create a `.example.env` file with exactly what you put below. 

You can then add a note in the README indicating that the user should copy the `.example.env` file to a `.env` and update any fields.